src/frontend/efiling-frontend/src/modules/helpers/creditCardAlerts.js

```diff
@@ -23,93 +23,86 @@ const registerCard = () => {
+    type="success"
+    styling="bcgov-success-background bcgov-no-padding-bottom"
+    element={
+      <p>
+        <span>
+          You have a valid Credit Card registered with your CSO account.
+        </span>
+        <br />
+        <span
+          onClick={() => registerCard()}
+          onKeyDown={() => registerCard()}
+          className="file-href"
+          role="button"
+          tabIndex={0}
+        >
+          Register a new Credit Card.
+        </span>
+      </p>
+    }
+  />
+);
 
-const noCreditCard = () => {
-  return (
-    <Alert
-      icon={<MdCreditCard size={32} />}
-      type="error"
-      styling="bcgov-error-background bcgov-no-padding-bottom"
-      element={
-        <p>
-          <span>
-            {sessionStorage.getItem("bamboraErrorExists") === "true" && (
-              <strong>Credit Card validation failed.&nbsp;</strong>
-            )}
-            You do not have a valid Credit Card registered with your CSO
-            account.
-          </span>
-          <br />
-          <span
-            onClick={() => registerCard()}
-            onKeyDown={() => registerCard()}
-            className="file-href"
-            role="button"
-            tabIndex={0}
-          >
-            Register a Credit Card now
-          </span>
-          &nbsp;to continue.
-        </p>
-      }
-    />
-  );
-};
+const noCreditCard = () => (
+  <Alert
+    icon={<MdCreditCard size={32} />}
+    type="error"
+    styling="bcgov-error-background bcgov-no-padding-bottom"
+    element={
+      <p>
+        <span>
+          {sessionStorage.getItem("bamboraErrorExists") === "true" && (
+            <strong>Credit Card validation failed.&nbsp;</strong>
+          )}
+          You do not have a valid Credit Card registered with your CSO account.
+        </span>
+        <br />
+        <span
+          onClick={() => registerCard()}
+          onKeyDown={() => registerCard()}
+          className="file-href"
+          role="button"
+          tabIndex={0}
+        >
+          Register a Credit Card now
+        </span>
+        &nbsp;to continue.
+      </p>
+    }
+  />
+);
 
-const failedUpdateCreditCard = () => {
-  return (
-    <Alert
-      icon={<MdCreditCard size={32} />}
-      type="warning"
-      styling="bcgov-warning-background bcgov-no-padding-bottom"
-      element={
-        <p>
-          <span>
-            <strong>Credit Card registration failed.&nbsp;</strong>
-            You have an existing Credit Card registered with your CSO account
-            and can proceed.
-          </span>
-          <br />
-          <span
-            onClick={() => registerCard()}
-            onKeyDown={() => registerCard()}
-            className="file-href"
-            role="button"
-            tabIndex={0}
-          >
-            Register a new Credit Card.
-          </span>
-        </p>
-      }
-    />
-  );
-};
+const failedUpdateCreditCard = () => (
+  <Alert
```

Comment:
I'm interested in the removal of the return. Give me a little insight?

Reply:
This was something that was caught by eslint after updating. When using Javascript arrow notation for functions, if your function is nothing but a return statement, you can eliminate the "return" and just return the expression directly. 

`const myFunction = () => {return (<div>Hello</div>)}`

is the same as doing

`const myFunction = () =>  (<div>Hello</div>)`

https://stackoverflow.com/a/52637049